tsconfig.build.json

```diff
@@ -1,6 +1,7 @@
 {
 	"extends": "@apify/tsconfig",
 	"compilerOptions": {
+		"types": ["node"],
```

Comment:
why is that? feels unrelated to the PR. your project should include `@types/node` in the dependencies, which is set up correctly in all of our templates